src/Components/InboxSummary/index.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react"
+import EmailSummary from "../EmailSummary"
+
+function InboxSummary () {
+
+const [emails, setEmails] = useState([])
+
+function getEmails() {
+    fetch('https://email-client-api.dev.io-academy.uk/emails')
+    .then(res => res.json())
+    .then(data => {
+    setEmails(data.data)
+  })
+}
+
+useEffect(getEmails, [])
```

Comment:
All of these lines should be indented once to show that they are within the InboxSummary function